lib/grammers-client/src/client/files.rs

```diff
@@ -9,16 +9,27 @@
 use crate::types::{Media, Uploaded};
 use crate::utils::generate_random_id;
 use crate::Client;
+use futures_util::future::try_join_all;
 use grammers_mtsender::InvocationError;
 use grammers_tl_types as tl;
-use std::io::SeekFrom;
-use std::path::Path;
-use tokio::fs;
-use tokio::io::{self, AsyncRead, AsyncReadExt as _, AsyncSeekExt as _, AsyncWriteExt as _};
+use std::{
+    io::SeekFrom,
+    path::Path,
+    sync::{
+        atomic::{AtomicI32, Ordering},
+        Arc,
+    },
+};
+use tokio::{
+    fs,
+    io::{self, AsyncRead, AsyncReadExt as _, AsyncSeekExt as _, AsyncWriteExt as _},
+    sync::Mutex,
+};
```

Comment:
The codebase is not using this import style anywhere else, so I think we should change it to be consistent to the non-nested form.